packages/python/plotly/plotly/graph_objs/_choroplethmapbox.py

```diff
@@ -3,6 +3,11 @@
 
 
 class Choroplethmapbox(_BaseTraceType):
+    print(
+        "*choroplethmapbox* trace is deprecated!",
+        "Please consider switching to the *choroplethmap* trace type and `map` subplots.",
+        "Learn more at: https://plotly.com/javascript/maplibre-migration/",
```

Comment:
All links should point to the Python documentation: https://plotly.com/python/mapbox-to-maplibre/